.buildkite/pipeline.yml

```diff
@@ -3,35 +3,35 @@
 
 steps:
   - label: hlint
-    branches: "!autodoc/master"
+    if: build.source != "schedule" && build.branch != "autodoc/master"
```

Comment:
Can we move this condition outside (define a variable with it) and reuse it across the steps?

Reply:
we can do something like `&not_scheduled_and_not_autodoc ` in the first occurrence, and use `*not_scheduled` later

Reply:
That's fine.